packages/boxed-expression-component/src/expressions/FunctionExpression/ParametersPopover.tsx

```diff
@@ -59,15 +61,31 @@ export const ParametersPopover: React.FunctionComponent<ParametersPopoverProps>
     });
   }, [setExpression]);
 
+  const addParameterButDoNotPropagatreEvent = useCallback(
```

Comment:
@jomarko I would unify `addParameterButDoNotPropagatreEvent` and `addParameter` methods and use it in both Buttons.

Reply:
done, please re-rereview the PR when you have a moment